plugins/modules/s3_bucket.py

```diff
@@ -2009,7 +2017,7 @@ def main():
         acl=dict(type="str", choices=["private", "public-read", "public-read-write", "authenticated-read"]),
         validate_bucket_name=dict(type="bool", default=True),
         dualstack=dict(default=False, type="bool"),
-        accelerate_enabled=dict(default=False, type="bool"),
```

Comment:
not sure if it's best path forward to change default value, might cause a breaking change

Reply:
@mandar242 a breaking change has been added to the changelog file